src/stories/containers/RecognizedDelegates/KeyStats/KeyDescriptionCards/PercentageTotalCard.tsx

```diff
@@ -0,0 +1,42 @@
+import styled from '@emotion/styled';
+import React from 'react';
+import KeyStatsCard from '../../GenericDelegateCard';
```

Comment:
You should rename the default export here

Reply:
it's done, in other commits